apps/obs-web/src/views/DebrieferAssessmentEditor.vue

```diff
@@ -0,0 +1,348 @@
+<template>
+    <div class="q-pa-md q-gutter-md">
+
+        <q-table
+          title="SECTIONS"
+          :data="sections"
+          :columns="sectionsColumns"
+          row-key="_id"
+          binary-state-sort
+          dense
+        >
+          <template v-slot:body="props">
+            <q-tr :props="props">
+              <q-td key="sections" :props="props">
+                {{ getArrayString(props.row.sections) }}
+                <q-popup-edit v-model="props.row.sections" buttons persistent @save="save(props.row)">
```

Comment:
There's an odd thing going on where if I click to edit one box, then click to edit another box, the previous box doesn't go away. It's a bit odd to see 5 edit boxes showing at once and then have to hit cancel to close all them. So probably showing one edit box at a time is better.

Reply:
fair enough - removing 'persistence' on edit dialogs.